docs/interpolation.rst

```diff
@@ -0,0 +1,17 @@
+.. _interpolation:
+
+Interpolation of IRFs
+=====================
+
+This module contains functions to interpolate from a set of IRFs for different
+condiditions to a new IRF.
+
+This can e.\,g. be used to interpolate IRFs for zenith angles of 20° and 40°
+to 30°.
```

Comment:
Hi @maxnoe 

I think it should me mentioned in the documentation that the interpolation is done over two dimensions (e.g. zenith and azimuth).

And sorry for forgetting the old compare_irf_cuts function - thx for fixing it here.

Reply:
It *could* be done using two dimensions, however, the functions allow arbitrary number of dimensions, right?

Reply:
is "IRFsaa" a special thing I don't know or a typo?

Reply:
typo :see_no_evil:

Reply:
Hi @maxnoe 
bulk of the functions is done in a way that it would allow an arbitrary number of dimensions, but the actual function used for the interpolation: griddata I think works only in 2D. Since this is the most typical use case, I think this is not a problem, but we can also consider substituting the griddata function with some other interpolation engine for higher number of dimensions.

Reply:
https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.griddata.html

No, griddata does arbitrary number of dimensions